### DISCUSSION TITLE
how to run multiple workers in one machine

### DISCUSSION THREAD

**Discussion by kenneth-1024** [filecoin-project/lotus#7081] [Storage Provider]
Hi all,
My machine has more than one GPUs, and I use it to run PC2 and C2.  Following the suggestion from [lotus-worker-co-location](https://docs.filecoin.io/mine/lotus/seal-workers/#lotus-worker-co-location), because lotus currently only supports a single GPU, I should "run workers in separate containers". 
And my question is how to do this? Does it mean run worker in docker?

Any help will be appreciated.

**Comment by LexLuthr** [comment]
@wcw84 Hi, were you able to run the workers on the same node? Worker co-location can be achieved using containers like docker. But, I would recommend running them separately if you are planning to run a miner to avoid delays in WindowPost due to resource utilisation.
